File: Domain/Product.cs
using System;


namespace Domain
{
    public class Product
    {
        
        public Guid ProductId{get; set;}

        public string ProductName{get; set;}
        public string Sector{get;set;}
        public string Brand{get;set;}

        public float ValueOfProduct{get; set;}

        public string ModelYear{get; set;}

        public int Quantity{get; set;}

        public string Description{get; set;}

        
    }
}

File: client-app/src/app/models/product.ts
export interface IProduct {
  productId: string;
  productName: string;
  sector: string;
  brand: string;
  valueOfProduct: number;
  modelYear: string;
  quantity: number;
  description: string;
}
export class ProductFormValues {
  productId: string = "" ;
  productName: string = '';
  sector: string = '';
  brand: string = '';
  valueOfProduct?: number ;
  modelYear: string = "";
  quantity?: number ;
  description: string = "";
  constructor(init?: any) {
    Object.assign(this, init);
  }
}
